Common.mk

```
@@ -425,6 +425,11 @@ SKIP_CHECKSUM_VALIDATION?=false
 IN_DOCKER_TARGETS=all-attributions all-attributions-checksums all-checksums attribution attribution-checksums binaries checksums clean clean-go-cache
 ####################################################
 
+#################### LOGGING #######################
+TARGET_START_LOG?=$(eval _START_TIME:=$(shell date +%s))\\n------------------- $(shell date +"%Y-%m-%dT%H:%M:%S%z") Starting $@ target ------------------- 
```

some of these targets are super quick, add tenths of a second?

```suggestion
TARGET_START_LOG?=$(eval _START_TIME:=$(shell date +%s))\\n------------------- $(shell date +"%Y-%m-%dT%H:%M:%S:%1N%z") Starting $@ target ------------------- 
```

---

hahah that sure is something